backend/battle/views.py

```diff
@@ -69,3 +69,20 @@ def get_queryset(self):
 class BattleDetail(DetailView):
     model = Battle
     template_name = "battle/battle_detail.html"
+
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()  # pylint: disable=attribute-defined-outside-init
+        context = self.get_context_data(object=self.object)
+        return self.render_to_response(context)
+
+    def get_context_data(self, **kwargs):
+        context = {}
+        if self.object:
+            context['object'] = self.object
+            context_object_name = self.get_context_object_name(self.object)
+            if context_object_name:
+                context[context_object_name] = self.object
+        team = Team.objects.filter(battle=self.object, trainer=self.request.user)
+        kwargs['team'] = team
+        context.update(kwargs)
```

Comment:
I think you can change this to `context['team'] = team`

Reply:
Thank you!